### DISCUSSION TITLE
Import a custom ttk creator theme from a local my_theme.py file

### DISCUSSION THREAD

**Discussion by lorenzolpandolfo** [israel-dryer/ttkbootstrap#590] [Q&A]
Hello everyone!
I am having some trouble to load a ttk creator custom theme. When I export it, it generates a theme.py file. But how do I load it to my Style object with its path?

I created my own theme `pytext.py`. The variable `ttkbootstrap.theme.user.USER_THEMES` recognize it, but only on my PC. I want to ship the theme.py file to my repository and load it with its path, so anyone who will run my tkinter app will have the pytext.py file to load it.

A intuitive way to do this would be like:
`style = Style(theme='/home/user/pytext/theme/pytext.py')`

Thank you!

**Comment by brobr** [comment]
See https://github.com/israel-dryer/ttkbootstrap/issues/697
